js/samples/advanced-csharp/Pages/Options.cshtml

```diff
@@ -0,0 +1,141 @@
+    <script type='text/javascript' src='https://ircdname.azureedge.net/immersivereadersdk/immersive-reader-sdk.1.2.0.js'></script>
+    <script type='text/javascript' src='~/js/helpers.js'></script>
+
+    <link href='~/css/styles.css' rel='stylesheet'>
+    <link href='~/css/buttons.css' rel='stylesheet'>
+
+    <script type='text/javascript'>
+        var Canary = '@Html.Raw(ViewData["Canary"])';
+    </script>
+
+    <style media='screen' type='text/css'>
+        #ContentArea {
+            display: block;
+            margin: 0 20%;
+            padding-top: 50px;
+            width: 60%;
+        }
+
+        .section {
+            border: 1px solid #cccccc;
+            border-radius: 4px;
+            box-sizing: border-box;
+            display: block;
+            margin-bottom: 20px;
+            padding: 20px 20px 10px 20px;
+            position: relative;
+            width: 100%;
+        }
+
+        #Cols {
+            box-sizing: inherit;
+            column-count: 1;
+            column-gap: 0;
+        }
+
+        h3 {
+            margin: 0;
+            margin-top: 10px;
+            padding: 0;
+        }
+
+        p {
+            line-height: 25px;
+        }
+
+        @@media screen and (min-width: 900px) {
+            #Cols {
+                column-gap: 20px;
+                column-count: 2;
+            }
+        }
+    </style>
+</head>
+<body class='body'>
+    <nav>
+        Examples:
+        <a href='/index'>Sections</a>
+        <a href='/document'>Document</a>
+        <a href='/multilang'>Multilingual Document</a>
+        <a href='/uilangs'>UI Language</a>
+        <a href='/math'>Math</a>
+        <a href='/worddoc'>Word Document</a>
+        <a href='/options' class='active'>Options</a>
+    </nav>
+
+    <main id='ContentArea'>
+        <h1>You can disable Immersive Reader features as desired!</h1>
+        <p>These are some of the features you can disable:</p>
+        <p><code>disableGrammar</code> Grammar features such as Syllables, Parts of speech, and Picture Dictionary will be disabled.</p>
+        <p><code>disableLanguageDetection</code> The Language Detection feature will be disabled. This will prevent Immersive Reader from automatically detecting the language of the supplied content.</p>
+        <p><code>disableTranslation</code> The Translation features will be disabled.</p>
+        <p><code>parent</code> Immersive reader is launched in the specified element, otherwise will be launched in the main document body.</p>
+        <h3>Examples:</h3><br />
+        <section class='section' id='SampledisableLanguageDetection'>
+            <div class='immersive-reader-button ir-button-small' onclick="handleLaunchImmersiveReader('disableLanguageDetection')"></div>
+            Specify the content language: <input id='langdisableLanguageDetection' value='en' /> (i.e 'fr', 'es', 'de', 'zh')
```

Comment:
suggest id of (adding underscore)

lang_* so its a bit easier to track from definition to usage

Reply:
done